### PR TITLE
Defuzzify polish translation 

### PR DESCRIPTION
**Pull request by nabijaczleweli**

### CODE REVIEW

File: po/pl.po [hunk 1]
```diff
@@ -1766,12 +1761,12 @@ msgid "Invalid number: %ld"
...
+#, c-format
 msgid "Invalid quad value: %s"
-msgstr "Błędna data względna: %s"
+msgstr "Błędna wartość pytania: %s"
 
 #: config/quad.c:132
-#, fuzzy, c-format
+#, c-format
 msgid "Invalid quad value: %ld"
 msgstr "Niewłaściwa wartość opcji %s: %ld"
```
Comment (flatcap) [inline po/pl.po:1771]: This translation has non-matching expandos: `%ld` vs `%s: %ld`

File: po/pl.po [hunk 2]
```diff
@@ -1798,14 +1793,14 @@ msgid "Type '%s' doesn't support operation '%s'"
...
 msgid "Invalid sort name: %s"
-msgstr "Błędna data względna: %s"
+msgstr "Błędna nazwa sortowania: %s"
 
 #: config/sort.c:161
-#, fuzzy, c-format
+#, c-format
 msgid "Invalid sort type: %ld"
-msgstr "Błędna data względna: %s"
+msgstr "Błędny typ sortowania: %s"
```
Comment (flatcap) [inline po/pl.po:1803]: Same here, `%ld` vs `%s`

**Comment by nabijaczleweli** [comment]
uhoh, meant to make this branch on my fork, sorry; either way

**Review by flatcap — CHANGES_REQUESTED** [review]
You can check your work locally using the command:
```sh
msgfmt --statistics -c -o /dev/null po/pl.po
```

Note: I don't speak Polish, so I can't really do much of a review :-)

As long as that command doesn't report any errors,
you are welcome to **push directly** to the **translate** branch.

**Comment by nabijaczleweli** [comment]
Oh, good catch, the fuzzy strings threw me off there. I re-reviewed this and the other lines under this light.

If so, I'll close this and push there directly, thanks!

**Pull request closed without merging**